### ISSUE TITLE
Implement Warmup

### ISSUE THREAD

**Issue by DeepBlockDeepak** [DeepBlockDeepak/my-neural-network#16]
Taken From [Super Study Guide](https://www.amazon.com/Super-Study-Guide-Algorithms-Structures/dp/B0B2TW6896):
## Warmup
A method for preventing early overfitting
## Idea
Use a lower learning rate **_ɑ_** during the first steps **_s_** to prevent model weights **_θ_** from being changed too dramatically due to noisy gradient estimates  **_∇ L_**
